Week-2/Session4-Advanced HTML/restaurantReviews/about.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Sono:wght@200&display=swap');
+        ul {
+            list-style-type: none;
+            margin: 0;
```

Comment:
move this to external styles

Reply:
@anilrao316 can you please provide some reference. I am not sure what I need to do here so.